Cargo.toml

```diff
@@ -4,6 +4,15 @@ version = "0.4.5"
 authors = ["M*C*O <mcofficer@gmx.de>"]
 edition = "2018"
 build = "build.rs"
+description = "Endless Sky Launcher"
+
+[package.metadata.bundle]
+name = "ESLauncher2"
+identifier = "me.mcofficer.eslauncher2"
```

Comment:
While we're at it, should we change the identifier and the author field to EndlessSkyCommunity? I just went with the default when i set up the project, but once it went public, being the sole maintainer was never the goal.

Reply:
Updated

Reply:
Ah, didn't see this thread earlier. The change to the bundle identifier makes sense, but given the work needed to get this project off the ground, I think it's appropriate to keep MCO in the authors field for the package lifetime.

If it's desired to feature contributors by name somehow, we could use readme badging, as was proposed for ES a few months ago.